concepts/loops/about.md

```diff
@@ -0,0 +1,178 @@
+# About
+
+In Java there are four looping constructs, two that are condition centric: `while` and `do-while` and the other two are iteration centric: `for` and `for-each`.
```

Comment:
Maybe it makes sense to rewrite this sentence to list the constructs in the order in which they are described below, so `for`, `for-each`, `while`, `do-while`.

Reply:
Done!